core/client/app/styles/components/power-select.css

```diff
@@ -106,3 +110,14 @@
     background: color(var(--blue) alpha(-85%));
     color: var(--darkgrey);
 }
+
+/*
+    HACK: ember-power-select has no separate class for the loading message
+    Issue: https://github.com/cibernox/ember-power-select/issues/479
+ */
+.ember-power-select-dropdown > .ember-power-select-options > .ember-power-select-option:first-of-type,
```

Comment:
The PR I submitted to ember-power-select has been merged and should be in a new release due this weekend so hopefully this particular piece of ugliness won't be in the codebase for too long.